src/JWT/Jwk/JwtWebKeysCollection.cs

```diff
@@ -0,0 +1,38 @@
+using System.Collections.Generic;
+using System.Linq;
+using JWT.Serializers;
+
+namespace JWT.Jwk
+{
+    public class JwtWebKeysCollection : IJwtWebKeysCollection
```

Comment:
Do we really have to roll a separate interface and implement it using a custom class?

Reply:
I want it to be flexible and allow users to have their own implementation, e.g. it could be querying from the database by `keyId`